crates/sui-node/src/lib.rs

```diff
@@ -468,6 +468,12 @@ impl SuiNode {
             ChainIdentifier::from(*genesis.checkpoint().digest()),
         );
 
+        replay_log!(
```

Comment:
This isn't even strictly needed, but added to ensure that things compile

Reply:
Thanks!